task.go

```diff
@@ -0,0 +1,21 @@
+package libcarina
+
+const (
+	resizeTaskType = "resize"
+)
+
+// ResizeInput is an input params for a resize task
+type ResizeInput struct {
```

Comment:
Let's change this to be unexported (lower case first letter). Since it isn't necessary for package consumers to use, let's not expose it and have it show show up in the package documentation.